src/main/scala/io/iohk/ethereum/NodeStatusHolder.scala

```diff
@@ -0,0 +1,72 @@
+package io.iohk.ethereum
+
+import java.net.InetSocketAddress
+
+import akka.actor.{ActorLogging, Props, Actor}
+import akka.util.ByteString
+import io.iohk.ethereum.network._
+import io.iohk.ethereum.utils.Config
+import org.spongycastle.crypto.AsymmetricCipherKeyPair
+import org.spongycastle.crypto.params.ECPublicKeyParameters
+import org.spongycastle.util.encoders.Hex
+
+class NodeStatusHolder(key: AsymmetricCipherKeyPair) extends Actor with ActorLogging {
+
```

Comment:
It will remove a few lines of code, if/when we have to change it back later - you can remind me I asked you to change it. ;)